ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalConsultation.cshtml.cs

```diff
@@ -9,29 +11,28 @@ namespace ApplyToBecomeInternal.Pages.TaskList.LegalRequirements
 {
 	public class LegalConsultationModel : LegalModelBase
 	{
-		public LegalConsultationModel(ILegalRequirementsRepository legalRequirementsRepository,
-			IAcademyConversionProjectRepository academyConversionProjectRepository) :
-			base(legalRequirementsRepository, academyConversionProjectRepository)
+		public LegalConsultationModel(IAcademyConversionProjectRepository academyConversionProjectRepository) :
+			base(academyConversionProjectRepository)
 		{
 		}
 
 		[BindProperty] public string Done { get; set; }
 
 		public void OnGet(int id)
 		{
-			Done = LegalRequirements.ConsultationDone.ToString();
+			Done = Requirements.ConsultationDone.ToString();
 		}
 
 		public async Task<IActionResult> OnPostAsync(int id)
 		{
-			LegalRequirements.ConsultationDone = Done switch
+			Requirements.ConsultationDone = Done switch
```

Comment:
I know you didn't write this. This could probably live in the base class. 
Or make a generic ToEnum extension method. Might save some screen space.